metal-k8s.yml

```diff
@@ -52,6 +52,7 @@
       tags: ['heapster']
     - role: kube_elasticsearch
       tags: ['elasticsearch']
+      when: metalk8s_elasticsearch_enabled|default(True)|bool
```

Comment:
This introduces a 'magic' var. Shouldn't this be set in some role's `defaults` with the appropriate default value?

Reply:
Yes indeed, variables behave differently in playbook, but now we have the default that stacks before the play to start. Two options:

* just moving default to kubernetes_elasticsearch would work
* having everything in the role (having main.yml that only include (conditionally) the real stuff